### DISCUSSION TITLE
New Category: Resource Pack Pack

### DISCUSSION THREAD

**Discussion by BrownTurtleBear** [modrinth/.github#156] [Feature requests]
A Brand new category never seen before. 
Like Modpacks but for resource packs. Users can choose the resource packs and their order to make combinations of resource packs that make a great experience. 
The name of the category can be different then the title ofc. Like... Packed Resources, Resource pack set.
It would make modrinth one of a kind

**Comment by triphora** [comment]
This can be done with modpacks though. Just add resource packs to a modpack.

**Reply by BrownTurtleBear** [reply]
Oh cool. But wouldnt the users get confused about which have mods and which don't? I still think if there should be a brand new category. Then more people would make the resource pack packs

**Reply by triphora** [reply]
Ultimately I don't think splitting it into its own project type would do much. It would still be underutilized and a fairly niche thing. It's not worth it to put a load of effort into creating a new project type that would barely be used.

P.S. Think of it this way: "modpacks" = "Modrinth packs", not necessarily "mod packs"

**Reply by BrownTurtleBear** [reply]
Hmm I see. I understand.
